wagn/lib/wagn/config/environments/development.rb

```diff
@@ -1,3 +1,9 @@
+
+
+Decko::Engine.configure do
+  config.cache_classes = false
+end
```

Comment:
It was already in the application (further down the same file).  I left it there, but if it's there alone, it doesn't work.